docker/nginx-container/docker-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
```

Comment:
Nice, but keep this as your local config as these settings are reasonably defaults in most cases.

Reply:
It would be helpful to provide origin of source in comment.

Reply:
I have indicated the source of this file in the comment. I agree. Thanks

It was necessary to add a condition check to see if the first command line argument matches `/bin/bash` Otherwise, other necessary nginx scripts will not be triggered and started.